docs/index.md

```diff
@@ -2,23 +2,22 @@
+Authentic Chained Data Containers (ACDC) let formal assertions about data be chained together such that the derivation of any particular datum is traceable back to its source.
 
 ## Motivation
 
-We need a way to chain authentic data together, allowing its provenance to be traced. This is valuable in many use cases:
+Digital signatures let us prove who said what. This is extremely valuable. However, it is often the case that a party who says something is just repeating data that they themselves received from someone else. This indirection may change the reliability of the data, as well as its effects in complex knowledge graphs. Many real-world use cases require us to model this indirection. For example:
 
 * Supply chain, where a valid transfer of custody downstream depends on the validity of all upstream handoffs.
-* Delegation, where a delegate's privileges derive from a chain of authorizations that must extend back to a source that is empowered.
-* Citation of sources (in art, in journalism, in academia, or in credential issuance), where an author wants to clarify that a particular assertion originates elsewhere. This allows the assertion to acquire (or lose) gravitas independent of the reputation of the author who includes it. It also allows analysis of license compliance.
+* Delegation, where a delegate's privileges derive from a chain of authorizations that must extend back to a valid root authority.
+* Statistical inference and legal judgements, where multiple sources of data may assert contradictory facts, and the ability to weigh cumulative evidence is vital.
+* Citation of sources (in art, in journalism, in academia, or in credential issuance), where an author wants to clarify that a particular assertion originates elsewhere. This allows the assertion to acquire (or lose) gravitas independent of the reputation of the author who repeats it, and it lets reusers of data be held accountable for compliance with data sharing constraints.
 
-The last of these examples deserves special comment. There is a tension between the decentralization that we want in a verifiable credential ecosystem, and the way that trust tends to centralize because knowledge and reputation are unevenly distributed. We want anyone to be able to attest to anything they like--but we know that verifiers care very much about the reputation of the parties that make those attestations.
+The last of these examples deserves special comment. There is a tension between the decentralization that we want in a verifiable data ecosystem, and the way that trust tends to centralize because knowledge and reputation are unevenly distributed. We want anyone to be able to attest to anything they like &mdash; but we know that verifiers care deeply about the reputation of the parties that make those attestations.
```

Comment:
should we use instead of verifiable data -> authentic data ?
verifiable is not explicit which verification we are talking here about.